src/bitcoin-cli.cpp

```diff
@@ -1175,6 +1175,12 @@ static int CommandLineRPC(int argc, char *argv[])
                 fputc('\n', stdout);
             }
         }
+        if (gArgs.IsArgSet("-getinfo") +
+            gArgs.GetBoolArg("-netinfo", false) +
+            gArgs.GetBoolArg("-generate", false) +
+            gArgs.GetBoolArg("-addrinfo", false) > 1) {
```

Comment:
The implicit conversions from bool to int seem safe (at least at first glance, https://en.cppreference.com/w/cpp/language/implicit_conversion), but I didn't get a change to dig into C++ standard docs to check.  It is a tiny bit unintuitive for a reader, but this doesn't seem too cryptic and seems cleaner than the separate `if` statements and temp variable used in 27815.

Reply:
Thanks for pointing this out,  the reference specifies an implicit conversion from bool to int is unsafe for `T::operator bool() const;`, but for this scenario, since we are not using a custom class it could be safe